README.md

```diff
@@ -1,2 +1,5 @@
 # prometheus-config-controller
+
+[![CircleCI](https://circleci.com/gh/giantswarm/prometheus-config-controller.svg?style=svg)](https://circleci.com/gh/giantswarm/prometheus-config-controller) [![Docker Repository on Quay](https://quay.io/repository/giantswarm/prometheus-config-controller/status "Docker Repository on Quay")](https://quay.io/repository/giantswarm/prometheus-config-controller)
+
```

Comment:
In other projects the badges are at the top of the readme and the headline is below them.

Reply:
lol cool, i'll update